### ISSUE TITLE
About the split of datasets

### ISSUE THREAD

**Issue by cshizhe** [lzrobots/LearningToCompare_ZSL#9]
Hi Li,

I am confused about the dataset split in the training codes.
In the [training codes]((https://github.com/lzrobots/LearningToCompare_ZSL/blob/master/AwA2_RN.py#L84)), the train/val splits provided by the GBU paper are combined as the training set, and the testing set is used as validation set to select the best model. Such setups cannot reliably indicate the real performance on the testing set.
I am wondering is this the standard setup for ZSL on these datasets?

Best,
Shizhe
